src/app/services/validation.ts

```diff
@@ -18,7 +18,16 @@ export function zeroOrLess(possibleNumber?: number): boolean {
 }
 
 export function validateName(userName?: string | null) {
-  return userName && userName.length > 0 && userName.length <= 255 && !userName.includes("*");
+  const forbiddenWords = ["https", "www"];
+  const validPattern = /^[\p{L}\-'`! ]+$/u;
```

Comment:
As with the API changes, this change permits backticks and exclamation marks (shouldn't be needed for a name?)

Reply:
Back ticks are invalid now, ! & _ are allowed